automation/test.sh

```diff
@@ -140,17 +140,7 @@ fi
 
 kubectl() { cluster/kubectl.sh "$@"; }
 
-
-# If run on CI use random kubevirt system-namespaces
-if [ -n "${JOB_NAME}" ]; then
-  export NAMESPACE="kubevirt-system-$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 10 | head -n 1)"
-  cat >hack/config-local.sh <<EOF
-namespace=${NAMESPACE}
-EOF
-else
-  export NAMESPACE="${NAMESPACE:-kubevirt}"
-fi
-
+export NAMESPACE="${NAMESPACE:-kubevirt}"
```

Comment:
I thought the randomized namespace was partially to ensure we never make any assumptions that the infrastructure lives in the "kubevirt" namespace.

Reply:
Yes it should. It broke when we switched to stdci because the env variable does not exist anymore I need to fix it in an extra PR.

Reply:
I created #2223 to track it.